pkg/config/legacy/tests/config.py

```diff
@@ -6,22 +6,16 @@
 # stdlib
 import ConfigParser
 from cStringIO import StringIO
-import glob
-import imp
-import inspect
-import itertools
 import logging
 import logging.config
 import logging.handlers
-from optparse import OptionParser, Values
 import os
-import platform
 import re
 from socket import gaierror, gethostbyname
 import string
 import sys
-import traceback
 from urlparse import urlparse
+from urllib.request import getproxies
```

Comment:
I am guessing this is the missing `getproxies` function from context, please double-check